### ISSUE TITLE
Anybody know how to do this with React Native 60

### ISSUE THREAD

**Issue by jrwpatterson** [dmitry-blackwave/adyen-react-native#14]
Hey I'm upgrading to React Native 60 and this library was working following the advice with 59... but with the new autolinking either I get the old null module or if I link it the legacy way it can't find the RCTBridge.h file.

**Comment by voslartomas** [comment]
@jrwpatterson have you solved this? I can see you have a fork of this lib.

**Comment by jrwpatterson** [comment]
Couldn’t work it out.... eventually dumped the swift file into project bridging header and that for some reason works with RN61 (release candidate 3) they seem to have sorted some swift stuff.

**Comment by voslartomas** [comment]
Ok and what about previous version 0.59.* was that working for you? What is the reason you upgrade?

**Comment by jrwpatterson** [comment]
Yes it was working for us... we needed the upgrade for the Facebook sdk

**Comment by voslartomas** [comment]
Could you please help me I am on 0.59.* version but still can't get it working. I am stuck on pod install 

`AdyenReactNative (from '../node_modules/adyen-react-native') dependency were found, but they required a higher minimum deployment target`

**Comment by jrwpatterson** [comment]
When I had that you need to raise the pod min version to 11.3

**Comment by voslartomas** [comment]
Weird, I have platform :ios, '11.3' specified, but still getting this error. Have you been on version 1.3.0 of adyen-react-native?

**Comment by voslartomas** [comment]
Sry, my bad, it was wrong Podfile, I had version 9.0 on different position. Thanks

**Comment by voslartomas** [comment]
One more thing, I can't find files like `Adyen.Framework,AdyenInternal.framework and AdyenInternal.bundle` in Products folder, it seems it was not generated. Did you follow official README tutorial? @jrwpatterson

**Comment by voslartomas** [comment]
@jrwpatterson Hey, so we migratet to RN 0.61.3-rc3 as well, could please explain what did you mean by "swift file into project bridging header" ? And did you just follow official readme?

**Comment by milansedliak** [comment]
Experiencing the same issue here. RN 0.61.3, pod install doesn't create the mentioned framework files under Pods/Products Is there some other way how to compile them?

**Comment by jrwpatterson** [comment]
sorry been on holiday! I did I had to build them... now I have issues because xcode 11 is a different swift version... I hate this stuff!

**Comment by mkharibalaji** [comment]
Hi Guy's,
          Sorry for the late response.I was busy with quite other work.Here is how it works.
update the AdyenReactNative.podspec file with this and do a 'pod install' it should work.

``
require 'json'
package = JSON.parse(File.read('package.json'))

Pod::Spec.new do |s|

  s.name         = "AdyenReactNative"
  s.version      = package["version"]
  s.summary      = package["description"]

  s.homepage     = "https://github.com/dmitry-blackwave/adyen-react-native#readme"
  s.license      = "MIT"
  s.author       = { "author" => "dmitry@belov.dev" }

  s.platform       = :ios, "11.0"
  s.source         = { :git => "https://github.com/author/AdyenReactNative.git" }
  s.source_files   = "ios/*.{h,m,swift}"
  s.requires_arc   = true

  s.dependency "React"
  s.dependency "Adyen","2.8.5"
  s.dependency "Adyen/ApplePay","2.8.5"

end
``

In the meanwhile I will raise a PR for this.

**Comment by mkharibalaji** [comment]
@jrwpatterson - What kind of issue are you facing with xcode 11 ?Can you please create a separate issue to trackit...
